x-pack/plugins/alerting/common/routes/framework/apis/health/schemas/v1.ts

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { schema } from '@kbn/config-schema';
+
+const alertingFrameworkHealthSchemaObject = {
+  status: schema.oneOf([schema.literal('ok'), schema.literal('warn'), schema.literal('error')]),
+  timestamp: schema.string(),
+};
+
+export const healthFrameworkResponseBodySchema = schema.object({
+  is_sufficiently_secure: schema.boolean({
+    meta: {
+      description: 'If false, security is enabled but TLS is not.',
```

Comment:
To my understanding the `is_sufficiently_secure` is calculated from the following:

1. The security plugin is enabled.
2. The `xpack.encryptedSavedObjects.encryptionKey` config is set in the `kibana.yml`
3. The API Keys are enabled in Elasticsearch.

Should we be more explicit or more high level? I am not sure how TLS is related to the above. Wdyt?

Reply:
I did just copy what the current description is in https://www.elastic.co/docs/api/doc/kibana/operation/operation-getalertinghealth and if we both don't exactly know I'd leave it as it is

Reply:
Ok!